File: 백준/9단계/소수찾기.py
N = int(input())
nums = list(map(int, input().split(' ')))

cnt = 0
for num in nums:
    error = 0
    if num > 1:
        for i in range(2, num):
            if num % i == 0:
                error += 1
        if error == 0:
            cnt += 1


# 소수는 1과 자기 자신으로 나눌때만 나누어 떨어지는 자연수

print(cnt)

File: 백준/9단계/소수찾기.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n')

const nums = input[1].split(' ').map(Number)
let cnt = 0

function isPrimeNum(num) {
    if (num < 2) {
        return false;
    }

    for (let i = 2; i < num; i++) {
        if (num % i == 0) {
            return false
        }
    }
    return true
}

for (let num of nums) {
    if (isPrimeNum(num) === true) {
        cnt += 1
    }
}

console.log(cnt)